packages/nextjs-mf/src/plugins/container/RemoveEagerModulesFromRuntimePlugin.ts

```diff
@@ -22,7 +34,7 @@ class RemoveEagerModulesFromRuntimePlugin {
       );
       return;
```

Comment:
The function 'apply' is not implemented in this diff. It's mentioned that it applies the plugin to the compiler. It would be better to check if the compiler is not null or undefined before applying the plugin.